yabt/buildcontext.py

```diff
@@ -135,6 +135,14 @@ def walk_target_deps_topological_order(self, target: Target):
             if dep_name in all_deps:
                 yield self.targets[dep_name]
 
+    def generate_direct_deps(self, target: Target):
+        yield from (self.targets[dep_name] for dep_name in target.deps)
```

Comment:
yield just pretend to run these with co-ruotines, and in practice it means you don't care about the order - right ?

Reply:
No coroutines here, just syntactic sugar for returning a generator from another generator. The order is maintained, but indeed in this case I don't care about it.

Reply:
gotcha.